.msggen.json

```diff
@@ -324,8 +324,7 @@
         "NewaddrAddresstype": {
             "all": 2,
             "bech32": 0,
-            "p2sh-segwit": 1,
-            "p2tr": 3
+            "p2tr": 1
```

Comment:
Never renumber protobuf fields. The numbering is how backwards and forwards compatibility is achieved. Renumbering leads to clients failing to parse messages. Please undo the numbering change, and if that leaves gaps, that's fine, we just cannot ever reuse the deprecated numbers.

Reply:
Mh, not this is not clear to me sorry! all these files are not autogenerated by msggen from https://github.com/ElementsProject/lightning/blob/master/doc/schemas/lightning-newaddr.json ?

Reply:
yes it's a bug in `msggen` im trying to fix rn